benchmark/Streamly/Benchmark/Data/Stream.hs

```diff
@@ -1,6 +1,6 @@
 -- |
--- Module      : Serial
--- Copyright   : (c) 2018 Composewell Technologies
+-- Module      : Data.Stream
+-- Copyright   : (c) 2022 Composewell Technologies
```

Comment:
Copyright year means "starting from that year", it should remain 2018 because the original code was written in that year.